### DISCUSSION TITLE
scope_info_client cleanup

### DISCUSSION THREAD

**Discussion by truthz03** [Mimetis/Dotmim.Sync#1259] [Q&A] [answered]
Hi,

I started to use this amazing library today but there are some things which are not clear yet.
As a client I use my Blazor Webassembly app and so far it seems to work but I recognized that the scope_info_client table always gets a new row when I reload my app.

Is there some kind of cleanup on the scope_info_client table or will this table grow and grow?

Thanks Thomas

**Comment by Mimetis** [comment]
Hi @truthz03 

I guess you are talking about the `scope_info_client` in the server database ?

If yes, then each row is associated with a client database.
Every time a new client database is registered on the server, a new row is added

Looking at your behavior, it seems that your blazor application is re-creating a new database each time it's launched ?

**Reply by truthz03** [reply]
Hi @Mimetis 

thanks for your answer.
Yes I mean the server database and yes the client always creates a new database.

I now get it working to persist the database inside the browser, therefore it is not a big problem now but maybe I have to cleanup the scope_info_clients table after some time.

Is it save to clear the table?
I tried to clear it and it seems to work (added the needed rows again if the client triggers a sync) but I'm not sure

**Reply by Mimetis** [reply, marked as answer]
you can, but if you are cleaning a client database that exists, the next sync will be a full sync from scratch
